client/src/components/common/Card.js

```diff
@@ -126,7 +127,7 @@ const useStyles = makeStyles((theme) => ({
     color: Colors.black
   },
   searchInput: {
-    margin: "4px 0",
+    margin: "2px 0",
```

Comment:
Let always use `theme.spacing`. To get `2px` you need to use `theme.spacing(1/3)`.

Reply:
theme.spacing(1) is 8px actually. For 2px we need to divide it by 4.